packages/react-next/src/components/Pivot/Pivot.tsx

```diff
@@ -1,8 +1,49 @@
 import * as React from 'react';
-import { styled } from '../../Utilities';
-import { IPivotProps, IPivotStyleProps, IPivotStyles } from './Pivot.types';
+import { styled, memoizeFunction, css } from '../../Utilities';
+import { IPivotProps, IPivotStyleProps, IPivotStyles, PivotLinkSizeType, PivotLinkFormatType } from './Pivot.types';
 import { PivotBase } from './Pivot.base';
-import { getStyles } from './Pivot.styles';
+import { getGlobalClassNames, ITheme } from '../../Styling';
+import * as classes from './Pivot.scss';
+
+const GlobalClassNames = {
+  count: 'ms-Pivot-count',
+  icon: 'ms-Pivot-icon',
+  linkIsSelected: 'is-selected',
+  link: 'ms-Pivot-link',
+  linkContent: 'ms-Pivot-linkContent',
+  root: 'ms-Pivot',
+  rootIsLarge: 'ms-Pivot--large',
+  rootIsTabs: 'ms-Pivot--tabs',
+  text: 'ms-Pivot-text',
+};
+
+const getStaticStylesMemoized = memoizeFunction(
+  (theme: ITheme, className?: string, linkSize?: PivotLinkSizeType, linkFormat?: PivotLinkFormatType) => {
+    const globalClassNames = getGlobalClassNames(GlobalClassNames, theme);
+
+    const rootModifiers = [
+      linkSize === 'large' && css(classes.linkSize_large, globalClassNames.rootIsLarge),
```

Comment:
@dzearing is there a preferred way to add these modifier classes? I.e. a way to translate `linkSize === 'large' ` into `classes.linkSize_large`

Reply:
Talked offline, but am going to try and make this cleaner.